src/ontology/mondo-edit.obo

```diff
@@ -371316,9 +371316,11 @@ xref: Orphanet:94063 {source="MONDO:equivalentTo"}
 xref: SCTID:719046005 {source="MONDO:equivalentTo"}
 xref: UMLS:C4305140 {source="MONDO:equivalentTo"}
 xref: UMLS:CN206727 {source="MONDO:equivalentTo"}
-is_a: MONDO:0000508 {source="Orphanet:94063"} ! syndromic intellectual disability
+is_a: MONDO:0000761 {source="Orphanet:94063"} ! syndrome caused by partial chromosomal deletion
+is_a: MONDO:0015160 {source="https://www.rarechromo.org/media/information/Chromosome%2012/12q14%20microdeletions%20FTNW.pdf"} ! multiple congenital anomalies/dysmorphic syndrome-variable intellectual disability syndrome
```

Comment:
as above, we shouldn't assert this parent (which comes from Orphanet), as this is not a parent in Orphanet, and I'm not clear this should be modeled this way.

In general, our preference is not to classify additional terms under grouping terms from Orphanet, such as this.